add Mint discs

mrjabba opened the pull request:
Not positive on these, 
gathered from https://mintdiscs.com and my experience with most of them. 
Cheers!

mrjabba committed:
add Mint discs

Not positive on these, gathered from https://mintdiscs.com and my experience with most of them. Cheers!

Daegalus commented:
So, Eternal is Proton (Same column as Champion), and Sublime is Plasma (Same column as GStar). You got the Apex correct for Neutron.

So if you can adjust those, I'll merge it.

mrjabba committed:
pr feedback

I think I got this right. Thanks for the fix! I haven't used the in browser PR stuff in a while :-)
